projects/packages/blaze/src/class-blaze.php

```diff
@@ -56,32 +56,66 @@ public function register() {
 	}
 
 	/**
-	 * Determines if criteria is met to enable Blaze features.
+	 * Check the WordPress.com REST API
+	 * to ensure that the site supports the Blaze feature.
+	 *
+	 * @param int $blog_id The blog ID to check.
 	 *
-	 * @todo - Get response from API if requirements are met on the wpcom-side.
+	 * @return bool
+	 */
+	private static function site_supports_blaze( $blog_id ) {
```

Comment:
Why private?

Reply:
Good point. Done in 347c4a42f0718bbea00fbe2007ec33d4c04c5917